test/integration/features/addons/addon-registry-route.feature

```diff
@@ -50,8 +63,8 @@ so that user can able to push their container images directly to registry and de
       And Minishift should have state "Running"
       And executing "minishift addon apply registry-route" succeeds
      Then with up to "60" retries with wait period of "3" seconds container name "registry_docker-registry" should be "running"
-      And with up to "60" retries with wait period of "1" second command "curl -ik https://docker-registry-default.$(minishift ip).nip.io" output should contain "200 OK"
-      And executing "docker login -u developer -p `oc whoami -t` docker-registry-default.$(minishift ip).nip.io" in host shell succeeds
+      And with up to "60" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "docker-registry" in namespace "default" is equal to "200"
```

Comment:
@agajdosi where is the implementation of this step I can just see the step but not able to figure out where we have implemented it.

Reply:
@praveenkumar It is implemented here: https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L681

Reply:
@agajdosi why don't then use https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L677 one since we know the URI already `docker-registry-default.$(minishift ip).nip.io` right? this will make this string also more readable I hope.

Reply:
@praveenkumar Scenario variables are currently only working within steps which executes commands. I cannot apply those to on https://github.com/minishift/minishift/blob/master/test/integration/testsuite/testsuite.go#L677.